.readthedocs.yml

```diff
@@ -0,0 +1,17 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+sphinx:
+  builder: html
+
+python:
+   version: 3.7
```

Comment:
YES!

Reply:
this is also how I know that the yml is working - it overrides the default config of 2.7 in the web GUI :)